package.json

```diff
@@ -3,14 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "compile": "rm -rf bin lib && ./node_modules/.bin/babel -s -d lib src/lib && ./node_modules/.bin/babel -s -d bin src/bin",
+    "install": "npm run compile",
+    "pretest": "npm run compile",
     "test": "test/runtests.sh"
   },
+  "babel": {
+    "sourceMaps": true,
+    "optional": [
+      "runtime"
+    ],
+    "stage": 1
+  },
```

Comment:
Why put it here? If we were publishing this to npm we wouldn't require consumers to use babel... Isn't that what this is for?
I suspect it's supported here for people who load babel modules using browserify w. transformer, or in a  node project...

I might be wrong here... You could certainly keep using `.babelrc`, but I'm not sure what is right...